pubtools/pulplib/_impl/model/__init__.py

```diff
@@ -1,8 +1,4 @@
-from .common import (
```

Comment:
unrelated change

Reply:
It's done by black automatically, not sure why.

Reply:
can you then fix it please?

Reply:
sure, let me try.